### DISCUSSION TITLE
Trouble running app

### DISCUSSION THREAD

**Discussion by austinreavers** [danny-avila/LibreChat#352] [Troubleshooting]
npm ERR! Cannot read properties of undefined (reading 'stdin')

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2023-05-22T04_50_42_383Z-debug-0.log.    

I keep getting this everytime I try and run the app

**Comment by fuegovic** [comment]
One possible solution is to install the latest version of node from this link:
https://nodejs.org/en/download/current
